.jules/bin/build_index.py

```
@@ -0,0 +1,60 @@
+import os
```

Ruff (0.15.9)

[warning] The file is executable but no shebang is present

(EXE002)